src/js/CloudExplorer.jsx

```diff
@@ -68,7 +68,7 @@ export default class CloudExplorer extends React.Component {
       this.ls();
     })
     .catch((e) => {
-      if (oldProps && oldProps.path) {
+      if (oldProps && oldProps.path && oldProps.path.length != 0) {
```

Comment:
I don't understand why you need to test that. And if you do, you should use `!==` which is type safe

Reply:
Indeed my first option was delete the whole **if** sentece, but I'm not sure why this senteces is here, so I choice modify, and avoid a back to a blank service. If it is executed then, when the error is processed, the original service is lost and the login process cannot restart  when press OK button in modal dialog. 
UnifileSerice is not modified in this request, just Cloudexplorer, as I explained at first. What happened is that I mixed another pull request.... :(